timestamper-wrapper/timestamperWrapper.groovy

```diff
@@ -0,0 +1,17 @@
+// This shows a simple build wrapper example, using the Timestamper plugin.
+node {
+    // This is the current syntax for invoking a build wrapper, naming the class.
+    wrap([$class: 'TimestamperBuildWrapper']) {
```

Comment:
Just curious, from a Groovy perspective can you implement this as an inline function like one would do normally in Groovy, e.g.

```groovy
void timestamper(Closure nested) {
  wrap([$class: 'TimestamperBuildWrapper']) {
    nested.call()
  }
}

node {
  timestamper {
    /* etc */
  }
}
```